cli/src/semgrep/run_scan.py

```diff
@@ -300,6 +300,16 @@ def run_rules(
     )
 
 
+# This is used for testing and comparing with osemgrep.
+def list_targets_and_exit(target_manager: TargetManager, product: out.Product) -> None:
+    targets = target_manager.get_files_for_language(None, product)
```

Comment:
We pass None here, but the type says Union[Language, Ecosystem], so how mypy allows this?
should be Optional[Union[Language,Ecosystem]] no?